services/user/Tokens/include/services/user/tokenTables.hpp

```diff
@@ -92,10 +91,13 @@ namespace UserService
       BalanceKey key;
       uint64_t   balance;
 
+      auto byUserToken() const { return key.account; }
```

Comment:
Secondary indexes need to be unique. The `Table` implementation does not implicitly append the primary key.

Reply:
Ahh, I was wondering why we were using tuples elsewhere, I forgot that.